gui/packages/ubuntupro/lib/pages/landscape/landscape_page.dart

```diff
@@ -0,0 +1,316 @@
+import 'dart:io';
+
+import 'package:file_picker/file_picker.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:flutter_markdown/flutter_markdown.dart';
+import 'package:provider/provider.dart';
+import 'package:ubuntu_service/ubuntu_service.dart';
+import 'package:wizard_router/wizard_router.dart';
+
+import '/core/agent_api_client.dart';
+import '/pages/widgets/page_widgets.dart';
+import 'landscape_model.dart';
+
+class LandscapePage extends StatelessWidget {
+  const LandscapePage({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    final model = context.watch<LandscapeModel>();
+    final lang = AppLocalizations.of(context);
+    final linkStyle = MarkdownStyleSheet.fromTheme(
+      Theme.of(context).copyWith(
+        textTheme: DarkStyledLandingPage.textTheme.copyWith(
+          bodyMedium: DarkStyledLandingPage.textTheme.bodyMedium?.copyWith(
+            fontWeight: FontWeight.w100,
+          ),
+        ),
+      ),
+    ).copyWith(
+      a: const TextStyle(
+        decoration: TextDecoration.underline,
+      ),
+    );
+
+    return ColumnLandingPage(
+      svgAsset: 'assets/Landscape-tag.svg',
+      title: 'Landscape',
+      onNext: !(model.fqdnError || model.fileError != FileError.none)
```

Comment:
This condition could be a getter inside the model:

```dart
bool get hasError => model.fqdnError || model.fileError != FileError.none;
```

Then you could rewrite this portion more easily:
```suggestion
      onNext: !model.hasError
```